tests/devices/test_default_mixed.py

```diff
@@ -729,3 +729,15 @@ def test_nr_wires(self):
         """Tests that an error is raised if the device is initialized with more than 23 wires"""
         with pytest.raises(ValueError, match="This device does not currently"):
             qml.device("default.mixed", wires=24)
+
+    def test_analytic_deprecation(self):
+        """Tests if the kwarg `analytic` is used and displays error message.
+        """
+        msg = "The attribute `analytic` has been replaced by `shots=None`. "
+        msg += "Please use `shots=None` instead of `analytic=True`."
```

Comment:
```suggestion
        msg = "The analytic argument has been replaced by shots=None. "
        msg += "Please use shots=None instead of analytic=True."
```